### DISCUSSION TITLE
Given that r26 seems to be re-based off LLVM17, will it include flang-new ?

### DISCUSSION THREAD

**Discussion by martin-frbg** [android/ndk#1916] [General]
Title says it all - the rc1 changelog mentions the included LLVM now being based on LLVM17 development. Was this just a one-time merge, or is development tracking that of LLVM17, meaning the NDK could once again contain a native Fortran compiler ?
(I know some may see this as archaic or exotic, but lots of Fortran code exists in numerical libraries that all kinds of programs depend upon, and the ability to build it was lost when the NDK switched from GCC to LLVM years ago)

**Comment by DanAlbert** [comment]
No. The NDK is a C/C++ SDK.

> Was this just a one-time merge, or is development tracking that of LLVM17

The NDK has always (well, since r11 anyway) tracked upstream LLVM.

> meaning the NDK could once again contain a native Fortran compiler ?

Citation needed? No NDK I've ever seen has had a fortran compiler.

**Comment by martin-frbg** [comment]
Oops, you're right, I have conflated two unrelated build problems from around r11 in my recollection, sorry.  Thanks for the quick reply.
